Makefile

```diff
@@ -1,8 +1,8 @@
+SRC_DIRS=src,src/sets,src/core,src/queues,src/indexed,src/maps,src/utils
```

Comment:
Should we make the change to use rebuild regardless of whether we adopt opam_of_packagejson?